### PR TITLE
Ana/Script1

### PR DESCRIPTION
**Pull request by anamariabaciu**
De data asta sper ca am facut bine

### CODE REVIEW

File: Ana/Task1 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+cd "anamariab"
+#ls
+
+#4-5
+git switch -c ANAScript
+
+
+#6
+touch "$MY_NAME.txt"
+ code "$MY_NAME.txt"
```
Comment (nbiro) [inline Ana/Task1:17]: what is this for?

File: Ana/Task1 [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#7-8
+for i in {0..10}
+do
+    touch gitMessage$i
+    if [ $i == 5 ];then
+        echo "$MY_NAME : my first script" > gitMessage$i
+    fi
+done
+
+bash  "$MY_NAME.txt"
```
Comment (nbiro) [inline Ana/Task1:27]: why this?

**Review by nbiro — COMMENTED** [review]
looks pretty good, couple of comments. Also, the script committed '.txt' file, I was expecting something like 'ana.txt'

File: Ana/Task1 (hunk 2 above)
Comment (anamariabaciu) [reply Ana/Task1:27]: nu trebuia sa dam bash ca sa ruleze codul ?  (sau nu cred ca am inteles eu 100% ce trebuia )

Comment (nbiro) [reply Ana/Task1:27]: pai ana.txt contine cod? e empty...

File: TaskAna2/script2.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+do
+    rm "gitMessage$i"
+done
+}
+
+scriptV2(){
+
+touch gitMessage{0..9}
+echo "$MY_NAME: my first script" >> "$(find *5)"
+echo "AnaMaria" >> "$(find *7)"
```
Comment (nbiro) [inline TaskAna2/script2.sh:31]: what is this for?

File: TaskAna2/script2.sh [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+commitmessage="$(grep -l "." gitMessage* | xargs cat)"
+rm gitMessage{0..9}
+}
+AnaMaria="$1"
+ cd ~/home/projectx/Desktop/ausy_academy_2022/sandbox/anamariab
+
+git switch -c "AnaMaria"
```
Comment (nbiro) [inline TaskAna2/script2.sh:38]: you should switch to $AnaMaria

File: TaskAna2/script2.sh [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+scriptV2(){
+
+touch gitMessage{0..9}
+echo "$MY_NAME: my first script" >> "$(find *5)"
+echo "AnaMaria" >> "$(find *7)"
+commitmessage="$(grep -l "." gitMessage* | xargs cat)"
+rm gitMessage{0..9}
+}
+AnaMaria="$1"
```
Comment (nbiro) [inline TaskAna2/script2.sh:35]: rename this to a more meaningful name( eg branch_name)

File: TaskAna2/script2.sh [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+"1")
+    scriptV2
+;;
+*)
+    scriptV1
+;;
+esac
+git add ~/home/projectx/Desktop/ausy_academy_2022/sandbox/anamariab/TaskAna2
+git commit -m "$commitmessage"
+git push origin ANAScript
```
Comment (nbiro) [inline TaskAna2/script2.sh:54]: branch is not ANAScript anymore, it's the first command line argument(AnaMaria variable in current context)

File: TaskAna2/script2.sh [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+scriptV1(){
+
+for i in {0..9}
+do
+    touch "gitMessage$i"
+    if [ $i -eq 5 ]; then
+        echo "$MY_NAME: my first script" >> gitMessage5
+    fi
+    if [ $i -eq 7 ]; then
+        echo "AnaMaria" >> gitMessage7
```
Comment (nbiro) [inline TaskAna2/script2.sh:12]: you should write the branch name to this file

**Review by nbiro — COMMENTED** [review]
see my comments

**Pull request closed without merging**